test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -2599,13 +2593,13 @@ describe('ThirdPartyRegistry', function () {
       // Buy 1000 item slots
       await manaContract.approve(
         thirdPartyRegistryContract.address,
-        TIERS[3].price,
+        oneEther.mul(toBN('500')),
```

Comment:
Same here for the price

Reply:
Besides the `slotsToAddOrBuy` variable, there is a `priceOfSlotsToBuy` variable to accompany it. On custom cases, you can use `getPrice(someDifferentAmountOfItemsToBuy)` to get a different price for buying slots or approving mana.